1.9/monitoring/logging/logging-api-examples.md

```diff
@@ -4,7 +4,16 @@ feature_maturity: preview
 menu_order: 4
 ---
 
-The following examples exercise the Logging API using [Bash](https://www.gnu.org/software/bash/), [Curl](https://curl.haxx.se/), and [jq](https://stedolan.github.io/jq/).
+This topic provides common usage examples for the Logging API.
+
+**Prerequisites:**
+
+- [Bash](https://www.gnu.org/software/bash/)
+- [Curl](https://curl.haxx.se/)
+- [jq](https://stedolan.github.io/jq/)
+- [DC/OS](/docs/1.9/installing/)
+- [DC/OS CLI](/docs/1.9/cli/) must be installed, configured, and logged in.
+- Extract the `DCOS_URL` and `DCOS_AUTH_TOKEN` from the DC/OS CLI...
```

Comment:
Did you mean to end line 16 with `...`? I think you meant to merge the content from line 18 into the bullet.